messages/fr/app.php

```diff
@@ -10,643 +10,643 @@
-    'This battle was recorded with an outdated version of IkaLog. Please upgrade to the latest version.' => '',
-    'This data was totaled after normalization to 5 minute intervals for each battle.' => '',
-    'This document is under a <a href="http://creativecommons.org/licenses/by/4.0/deed.en">Creative Commons Attribution 4.0 International License</a>.' => '',
-    'This information will be public. Integration for "log in with twitter" can be done from the profile page.' => '',
-    'This Month (UTC)' => '',
-    'This option is provided for deleting an incorrectly-reported battle.' => '',
-    'This twitter account has already been integrated with another user.' => '',
-    'This website doesn\'t support Internet Explorer. Please use a modern browser, for example Chrome or Firefox.' => '',
-    'This website has color-blind support. Please check "Color-Blind Support" in the "Username/Guest" menu of the navbar to enable it.' => '',
-    'This website is an open source project. It is under the MIT License. The source code is available on GitHub.' => '',
-    'This website is an UNOFFICIAL SERVICE. It is not related to the Splatoon development team or Nintendo.' => '',
-    'This {attribute} is already in use.' => '',
+    'The estimated location may be inaccurate.' => 'La localisation peut-être imprécise.',
+    'The icon will be:' => 'L\'icône sera :',
+    'Their team\'s league power' => 'Ligue Power de leur équipe',
+    'Their team\'s splatfest power' => 'Splatfest Power de leur équipe',
+    'Their Team Count' => 'Progression de leur équipe',
+    'Their Team Score' => 'Score de leur équipe',
+    'Their Team Splatfest Power' => 'Splatfest Power de leur équipe',
+    'Themes from {name}' => 'Thèmes de {name}',
+    'The password will be encrypted.' => 'Le mot de passe sera chiffré.',
+    'The post will be in the language set here.' => 'Le post sera mis dans la langue mise ici.',
+    'There are no data.' => 'Il n\'y a pas de données',
+    'There is no user associated with the specified twitter account.' => 'Il n\'y a pas de joueur associé à ce compte Twitter',
+    'These battles were recorded with an outdated version of IkaLog. Please upgrade to the latest version.' => 'Ces matchs ont été enregistrés avec une version obsolète d\'IkaLog. Veuillez mettre l\'application à jour.',
+    'This battle was recorded with an outdated version of IkaLog. Please upgrade to the latest version.' => 'Ce match a été enregistré avec une version obsolète d\'IkaLog. Veuillez mettre l\'application à jour.',
+    'This data was totaled after normalization to 5 minute intervals for each battle.' => 'Ces données ont été totalisées après une normalisation de chaque match à 5 minutes d\'intervalle chacun.',
+    'This document is under a <a href="http://creativecommons.org/licenses/by/4.0/deed.en">Creative Commons Attribution 4.0 International License</a>.' => 'Ce document est sous la licence CC By 4.0.',
```

Comment:
I forgot to put the html : `"Ce document est sous la licence <a href="http://creativecommons.org/licenses/by/4.0/deed.en">CC By 4.0</a>."`

Reply:
Maybe you can use `https://creativecommons.org/licenses/by/4.0/deed.fr` instead of `.en`.
I think that would be kinder to French speakers.